lib/docsparser.js

```diff
@@ -201,7 +201,7 @@ DocsParser.prototype.format_function = function(name, args, retval, options) {
 
 DocsParser.prototype.get_function_return_type = function(name, retval) {
     // returns None for no return type. False meaning unknown, or a string
-    if(name.search(/[A-Z]/) > -1)
+    if(name.search(/[A-Z]/) === 0)
```

Comment:
^^^ Agreed, I think this is a bit more clear @emflores. I tweaked the commit and pushed it back up.